app/bt/AffectedUserFlow/index.tsx

```diff
@@ -19,40 +23,94 @@ const fade: StackCardStyleInterpolator = ({ current }) => ({
   cardStyle: { opacity: current.progress },
 });
 
-const SCREEN_OPTIONS = {
-  cardStyleInterpolator: CardStyleInterpolators.forHorizontalIOS,
-  cardStyle: {
-    backgroundColor: 'transparent', // prevent white flash on Android
-  },
-  headerShown: false,
+const ExportStack = (): JSX.Element => {
+  const BackButton = () => {
+    const navigation = useNavigation();
+    return (
+      <TouchableOpacity
+        onPress={() => navigation.goBack()}
+        style={styles.iconContainer}>
+        <SvgXml xml={Icons.BackArrow} color={Colors.quaternaryViolet} />
+      </TouchableOpacity>
+    );
+  };
+
+  const CloseButton = () => {
+    const navigation = useNavigation();
+    return (
+      <TouchableOpacity
+        onPress={() => navigation.navigate(Screens.Settings)}
+        style={styles.iconContainer}>
+        <SvgXml xml={Icons.Close} color={Colors.quaternaryViolet} />
+      </TouchableOpacity>
+    );
+  };
+
+  return (
+    <AffectedUserProvider>
+      <Stack.Navigator
+        mode='modal'
+        screenOptions={{
+          cardStyleInterpolator: fade,
+          cardStyle: {
+            backgroundColor: 'transparent', // prevent white flash on Android
+          },
+          headerStyle: {
+            height: 100,
+          },
+          gestureEnabled: false,
+        }}
+        initialRouteName={Screens.AffectedUserStart}>
+        <Stack.Screen
+          name={Screens.AffectedUserStart}
+          component={Start}
+          options={{
+            headerTransparent: true,
+            headerTitle: '',
+          }}
+        />
+        <Stack.Screen
+          name={Screens.AffectedUserCodeInput}
+          component={CodeInput}
+          options={{
+            headerTransparent: true,
+            headerLeft: () => {
+              return <BackButton />;
+            },
+            headerTitle: '',
+            headerRight: () => {
```

Comment:
Could we pass `CloseButton` directly here? would this take care of the `display-name` rule error?
We could also name the function:

``` typescript
headerRight: function closeButton() { return <CloseButton />  }
```